src/modules/auth/header.js

```diff
@@ -99,8 +99,8 @@ const mapStateToProps = state => {
 		else if (userinfo.username) {
 			username = userinfo.username;
 		}
-		else if (userinfo.preferred_username) {
-			username = userinfo.preferred_username;
+		else if (userinfo.username) {
```

Comment:
@Aringoaway You already have `userinfo.username` on line 99, so please, leave `preferred_username` here as a fallback with the TODO and console warn message about that `preferre_username` is deprecated and will be removed by the end of 2023.

```
else if (userinfo.preferred_username) {
	// TODO: Remove preferred_username by the end of 2023
	console.warn("preferred_username is deprecated")
	username = userinfo.preferred_username;
}
```